solidity/contracts/deposit/DepositLiquidation.sol

```diff
@@ -50,7 +51,7 @@ library DepositLiquidation {
 
         // Determine value of the lot in wei
         uint256 _satoshiPrice = _d.fetchBitcoinPrice();
-        uint256 _lotSizeSatoshis = _d.lotSizeSatoshis;
+        uint64 _lotSizeSatoshis = _d.lotSizeSatoshis;
         uint256 _lotValue = _lotSizeSatoshis.mul(_satoshiPrice);
```

Comment:
Do we need to cast here?

Reply:
no need, it should be implicit

Reply:
So `_lotSizeSatoshis` will automatically be upcast so that it doesn't overflow when multiplied? Is that something `SafeMath` gives us?

Reply:
`mul(uint256 a, uint256 b)`, so  `_lotSizeSatoshis` should be upcast here